### PR TITLE
Project Microverse - Module One

### PR DESCRIPTION
**Pull request by Aqurds**
Added the following file
-  README.md,
- index.html
- assets/css/style.css
- linters
- .gitignore
- test.md (ignored by .gitignore)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+node_modules
+
+test.md
```
Comment (braincee) [inline .gitignore:3]: - [x] Please I highly recommend adding `test.md` file with text `This file is ignored by git` per project requirements. 😄

**Review by braincee — CHANGES_REQUESTED** [review]
Hello @Aqurds 👋 , I am @braincee your Code Reviewer

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

HIGHLIGHTS
------

 - GitHub flow rightfully used ✅

REQUIRED CHANGES ♻️
------

- [x] As mentioned in the [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md), you are to set up [linters for HTML and CSS](https://github.com/microverseinc/linters-config). I highly recommend adding `hintrc` and `stylelintrc.json` to your project. 😃 
- [x] I also noticed that you have included the [Microverse Readme template](https://github.com/microverseinc/readme-template) but you didn't customize it to suit this project. This makes your project incomplete. Every section of the readme template is required with the exception of the FAQ section which is optional. 😄 
- [x] Please follow the steps I have listed below to customize it to suit this project:
       - Remove the Microverse logo. If you have your logo, you can use it but I would advise you 
          to remove it 
       - Please apply the right project title instead of leaving the project title empty.
         For the Tech Stack Section, Kindly remove the Server and Database subsections because 
         the technologies listed there are not relevant to this project. For the Client section, kindly 
         include HTML and CSS 👍
      - For the Key Features section, kindly highlight what this project is all about. This will make 
         it easy for those who go through this repository to know what was implemented by 
         reading the README.md file. 😉
       - For the Prerequisite Section, kindly add tools that are needed before you can clone this 
          repository on a local machine. Tools like:
          ◦ Installation of Git
          ◦ Installation of NPM
          ◦ A working browser
          ◦ Internet connection
          ◦ A code editor like Visual Studio Code
          ◦ Installation of Live Server plugging in Visual Studio Code
          ◦ Any other tool you can think about 💯
       • For the Setup section, kindly add instructions for cloning your repository on a local 
         machine 👍
       • For the Install section, kindly add instructions for setting up linters used in this project 👍
       • For the Usage section, include instructions for opening this project on a browser. For 
         example, opening the project by launching the Live Server and plugging in visual studio 
         code 
         or any other means that you know.
       - For the Future Features section include 2 or more future features.
 

- [x] Kindly implement all the changes listed above. It will make your README.md file look professional and it will give you an idea of how to set README.md files for most of the projects you will execute as a Microverse student.  



**_Please check the comments under the review._**

OPTIONAL CHANGES
--------
**_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @braincee in your question so I can receive the notification.

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

----

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (Aqurds) [reply .gitignore:3]: I have already added test.md file with the text and added this file in .gitignore so it can't be added in github repository.

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @Aqurds,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ Linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**